lib/install/management/delete_test.go

```diff
@@ -89,12 +89,37 @@ func TestDelete(t *testing.T) {
 		createAppliance(ctx, validator.Session, conf, installSettings, false, t)
 
 		testNewVCHFromCompute(input.ComputeResourcePath, input.DisplayName, validator, t)
+		//		testUpgrade(input.ComputeResourcePath, input.DisplayName, validator, installSettings, t) TODO: does not implement: CreateSnapshot_Task
```

Comment:
good point, but there is one TODO item at the end of the line.
I cannot un-comment it at this time for vcsim still does not have the function support for snapshot, and I want to leave some hints there for what we should test using vcsim.